.travis.yml

```diff
@@ -1,17 +1,21 @@
 sudo: true
 language: php
 php:
-  - 7.1
   - 7.2
-services:
-  - mysql
+  - 7.3
 
 matrix:
     fast_finish: true
+    allowed_failures:
```

Comment:
`allow_failures:`

Reply:
:man_facepalming:

Reply:
i bet  i copy/pasted that across every repo...

Reply:
:sob:

Reply:
😿